x-pack/plugins/cloud_security_posture/public/pages/configurations/findings_flyout/overview_tab.tsx

```diff
@@ -118,9 +118,9 @@ const getDetailsList = (
       defaultMessage: 'Index',
```

Comment:
Can you also
1. change this to `Data View` 
2. in the line above it change the id from `xpack.csp.findings.findingsFlyout.overviewTab.indexTitle` to `xpack.csp.findings.findingsFlyout.overviewTab.dataViewTitle`
3. after that run `node scripts/i18n_check.js --fix `

Reply:
Done.:)